### PR TITLE
Layout: Clean up `Au` conversion helper functions in the flexbox 

### PR DESCRIPTION
**Pull request by atbrakhi**
---

- [x] `./mach build -d` does not report any errors
- [x] `./mach test-tidy` does not report any errors

### CODE REVIEW

**Comment by atbrakhi** [comment]
marking as draft until results can be updated.

File: components/layout_2020/flow/inline.rs [hunk 1]
```diff
@@ -2405,6 +2405,7 @@ impl<'a> ContentSizesComputation<'a> {
             had_non_whitespace_content_yet: false,
             linebreaker: None,
             ending_inline_pbm_stack: Vec::new(),
-        }.traverse(inline_formatting_context)
+        }
+        .traverse(inline_formatting_context)
```
Comment (mrobinson) [inline components/layout_2020/flow/inline.rs:2409]: I think this change is accidental.

**Review by mrobinson — APPROVED** [review]
Looks good. Feel free to land this after removing the unintentional change. Thanks!

File: components/layout_2020/flow/inline.rs (hunk 1 above)
Comment (atbrakhi) [reply components/layout_2020/flow/inline.rs:2409]: This is `./mach fmt` doing. I noticed this in another change as well. Would you like me to revert it? But mach fmt will suggest it again.

Comment (mrobinson) [reply components/layout_2020/flow/inline.rs:2409]: Huh! Sorry. Nope it's fine to leave it as-is.

**Pull request merged**